_posts/release/2018-02-16-accumulo-1.7.4.md

```diff
@@ -0,0 +1,63 @@
+---
+title: Apache Accumulo 1.7.4
+draft: true
+---
+
+Apache Accumulo 1.7.4 is a maintenance release on the 1.7 version branch. This release contains changes from 46 issues, comprised of bug-fixes, 
+performance improvements, build quality improvements, and more. See [Jira][JIRA_174] for a complete list.
+
+Below are resources for this release:
+
+* [User Manual][user_manual] - In-depth developer and administrator documentation.
+* [Javadocs][javadoc] - Accumulo 1.7.4 API
+* [Examples][examples] - Code with corresponding readme files that give step by step instructions for running example code.
+
+Accumulo follows [Semantic Versioning][semver] [guidelines][guidelines].  This release is a  “[patch version][patch_version]”, which means that
+only backwards compatible bug fixes are introduced in this version. A bug fix is defined as an internal change that fixes incorrect behavior. Users
+of any previous 1.7.x release are strongly encouraged to update as soon as possible to benefit from the  bug fixes with very little concern in change
+of underlying functionality.  As always, the Accumulo developers take API compatibility very seriously and have invested much time to ensure that we
+meet the promises set forth to our users. Users of 1.6 or earlier that are seeking to upgrade to 1.7 should consider 1.7.4 as a starting point.
+
+## Major Changes
+
+## Other Notable Changes
+
+* [ACCUMULO-4817] - Update build plugins and parent POM
+* [ACCUMULO-4809] - Avoid blocking during session clean up
+* [ACCUMULO-4787] - Close input stream in AccumuloReplicaSystem
+* [ACCUMULO-4778] - Cache table name to id map
```

Comment:
This issue was reverted for 1.7.

Reply:
It's removed now. I was using the git history and didn't see the revert.  This pushed me to create a python script that generates release notes from the git history.  I pushed two more commits.  The first one contains the output generated by my script and the second the changes I made to create the release.